arch/arm/boot/dts/overlays/README

```diff
@@ -3683,6 +3683,7 @@ Params: at056tn53v1             Enable an Innolux 5.6in VGA TFT
         backlight-def-brightness
                                 Set the default brightness. Normal range 1-16.
                                 (default 16).
+        rotate                  Display rotation {0,90,180,270} (default 0)
```

Comment:
This also needs to go in the README for vc4-kms-dpi-generic and vc4-kms-kippah-7inch because they also include vc4-kms-dpi.dtsi.

Otherwise looks fine to me.

Reply:
vc4-kms-kippah-7inch seems to be depreciated in favor of vc4-kms-dpi-panel. I'll update vc4-kms-dpi-generic and resubmit though, thank you.

Reply:
Nevermind, I understand. Even though it's depreciated it's still a valid devicetree (for now). I'll update it too, thank you.

Reply:
I'd seen your firmware mod to support renaming to a new overlay with params, but hadn't seen you'd updated the README. Sorry, my bad.

Reply:
Correct. Add to vc4-kms-dpi-generic, but not to the vc4-kms-kippah-7inch.

Messages about dummy regulators are almost inevitable. You can create a regulator with ```compatible = "regulator-fixed";``` if you really felt like it, but it's not really necessary.

Reply:
I won't worry about fixed regulators then for now (I tried it and there's no gain really aside from one less thing in dmesg).

I just updated the pull with the README fixes. Let me know if there is anything else that's needed. Thank you.